rules/es6.js

```diff
@@ -23,5 +23,7 @@ module.exports = {
         // Do not use these features yet
         'es/no-nullish-coalescing-operators': 'error',
         'es/no-optional-chaining': 'error',
+
+        'no-invalid-this': 'error'
```

Comment:
```suggestion
        'no-invalid-this': 'error',
```
This will fail on lint :sweat_smile: 

Can you also add a comment here?